### PR TITLE
Update sila_java & tweak code to support server initiated connection

### PR DESCRIPTION
**Pull request by QTimort**
- sila_java now points to usc branch
- Update call execution to work with current sila_java
- Add connection type in server info
- Partially implement server listener

See todo in OrchestratorGui L138

I tested the changes with some commands available in the TestServer and the CloudTestServer available in the sila_java examples.

### CODE REVIEW

**Comment by QTimort** [comment]
@FlorianBauer you updated the ref to sila_java to latest master, however most of the changes, except the call listener are not available on master at the moment, you need to use the branch usc on sila_java if you want to use them.

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [submodule "extern/sila_java"]
 	path = extern/sila_java
 	url = https://gitlab.com/SiLA2/sila_java.git
+	branch = usc
```
Comment (QTimort) [inline .gitmodules]: @FlorianBauer remove this line, it should then work

Comment (QTimort) [reply .gitmodules]: The reference to sila_java is correct (110ec60ea9a49f3a00a1b0a49c9654365ec7025f) but it seems that the CI does not pull the right version of sila_java

**Comment by FlorianBauer** [comment]
`git submodule update --init --recursive` in the CI script did the trick. However, no clue why the test runner failed on the windows build. Anyway, I consider this PR now as mergable.

 @QTimort  Thanks for the PR and all the help.

**Pull request merged**